src/ripplegw/waveforms/IMRPhenomXAS_NRTidalv3.py

```
@@ -25,14 +25,14 @@
 
 
 def _gen_IMRPhenomXAS_NRTidalv3(
-    f: Array,
+    f: Float[Array, " n_freq"],
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Expected an expression

(F722)